earth_enterprise/src/common/quadtreepath.h

```diff
@@ -78,6 +78,10 @@ class QuadtreePath {
       && (path2.IsAncestorOf(path1) || path2 > path1);
   }
 
+  static inline bool OverlapsAtLevel(const QuadtreePath &path1, const QuadtreePath &path2, uint level) {
```

Comment:
I think we now know that this function is wrong, or at least misses some important edge cases. It may be wise to leave it in for posterity, but we could comment it out or add a comment indicating that it's not fully correct.

Reply:
Oops. I guess this was created as part of the previous investigation but isn't used now. I completely missed that. I'll go ahead and take it out.